keras/utils.py

```diff
@@ -14,16 +15,23 @@
 from tanda.tan import PretrainedTAN
 from tanda.transformer import PadCropTransformer
 
-
-def load_pretrained_tan(config_path, checkpoint_path, tfs, dims=[32, 32, 3]):
-    # Load config
-    with open(config_path, 'r') as f:
+def load_pretrained_tan(path):
+    # Load config dictionary from run log
+    with open(os.path.join(path, 'run_log.json'), 'r') as f:
         config = json.load(f)
+
+    # Load TFs
+    # Assume they are present in config['train_module'] as list called tfs
+    tfs = import_module(config['train_module']).tfs
```

Comment:
This seems like way too strong of an assumption. Any other ideas on how to load in TFs?

Reply:
Fixed in latest commit- we just pickle them using the `cloud` lib (a good logging / reproducibility step regardless), then just load them in; much simpler.  And all handled in the `train_scripts.py` file